user_tools/docs/index.md

```diff
@@ -55,56 +55,75 @@ The wrapper has the following deployment modes:
-|            |               |             |   bootstrap [ARGS]              |          |
-+------------+---------------+-------------+---------------------------------+----------+
++------------------+---------------+-------------+---------------------------------------+----------+
+| platform         | command       | deploy mode |              CLI                      |  version |
++==================+===============+=============+=======================================+==========+
+| EMR              | qualification | local       | spark_rapids_user_tools \             |  23.02+  |
+|                  |               |             |   emr qualification [ARGS]            |          |
+|                  |               +-------------+---------------------------------------+----------+
+|                  |               | serverless  | spark_rapids_user_tools \             |  23.02+  |
+|                  |               |             |   emr qualification [ARGS] \          |          |
+|                  |               |             |   --mode=serverless                   |          |
+|                  |               +-------------+---------------------------------------+----------+
+|                  |               | cluster     |           unsupported                 |    N/A   |
+|                  +---------------+-------------+---------------------------------------+----------+
+|                  | profiling     | local       |           unsupported                 |    N/A   |
+|                  |               +-------------+---------------------------------------+----------+
+|                  |               | serverless  |           unsupported                 |    N/A   |
+|                  |               +-------------+---------------------------------------+----------+
+|                  |               | cluster     |           unsupported                 |    N/A   |
+|                  +---------------+-------------+---------------------------------------+----------+
+|                  | bootstrap     | local       | spark_rapids_user_tools \             |  23.02+  |
+|                  |               |             |   emr bootstrap [ARGS]                |          |
+|                  |               +-------------+---------------------------------------+----------+
+|                  |               | serverless  |           unsupported                 |    N/A   |
+|                  |               +-------------+---------------------------------------+----------+
+|                  |               | cluster     | spark_rapids_user_tools \             |  23.02+  |
+|                  |               |             |   emr bootstrap [ARGS]                |          |
++------------------+---------------+-------------+---------------------------------------+----------+
+| Dataproc         | qualification | local       | spark_rapids_user_tools \             |  23.04+  |
```

Comment:
Since 23.02.1 has been released we should update the versions for Dataproc from 23.04+ to 23.02.1+
For Databricks, it should stay 23.4+ because we haven't trigger a release yet.